events/app/views/components/event-promo/event-promo.njk

```diff
@@ -3,11 +3,39 @@
     {% componentV2 'image', model.image, {}, {lazyload: true, defaultSize: data.defaultSize} %}
```

Comment:
Styles for this file are here: https://github.com/wellcometrust/wellcomecollection.org/pull/1918/files#diff-3ee3e9325652c89d1899c09f05ffdff2R1 – what's the best order to merge in?

Reply:
@davidpmccormick https://github.com/wellcometrust/wellcomecollection.org/pull/1918#pullrequestreview-82457451